src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinActivityConversionExtensions.cs

```diff
@@ -47,6 +47,46 @@ internal static ZipkinSpan ToZipkinSpan(this Activity activity, ZipkinEndpoint l
 
             activity.EnumerateTags(ref tagState);
 
+            if (activity.Status != ActivityStatusCode.Unset)
+            {
+                PooledList<KeyValuePair<string, object>>.Add(
+                    ref tagState.Tags,
+                    new KeyValuePair<string, object>(
+                        SpanAttributeConstants.StatusCodeKey,
+                        StatusHelper.GetTagValueForActivityStatusCode(activity.Status)));
```

Comment:
I don't think the `GetTagValueForActivityStatusCode` is needed here. There is only 2 possible outcomes - OK or ERROR. A simple if..else should do the job, and it'd be faster than switch..case.

Reply:
I understand your point; but wouldn't the current implementation have better consistency with `GetTagValueForStatusCode` and `GetStatusCodeForTagValue`?

Reply:
Let speed be our guide :)